src/api/alchemy-wallet.ts

```diff
@@ -1,3 +1,105 @@
+import {
+  BlockTag,
+  FeeData,
+  Provider,
+  TransactionRequest,
+  TransactionResponse
+} from '@ethersproject/abstract-provider';
 import { Wallet as EthersWallet } from '@ethersproject/wallet';
+import { BytesLike } from '@ethersproject/bytes';
+import { ExternallyOwnedAccount } from '@ethersproject/abstract-signer';
+import { SigningKey } from '@ethersproject/signing-key';
+import { Deferrable } from '@ethersproject/properties';
+import { Alchemy } from './alchemy';
+import { BigNumber } from '@ethersproject/bignumber';
+export class Wallet extends EthersWallet {
```

Comment:
can you add doc comments on the top level describing what the differences vs. regular `Wallet` exported by ethers is?

Reply:
Done!